panther_analysis_tool/main.py

```diff
@@ -272,12 +272,14 @@ def test_analysis(args: argparse.Namespace) -> Tuple[int, list]:
     logging.info('Testing analysis packs in %s\n', args.path)
 
     # First classify each file, always include globals and data models location
-    data_models, global_analysis, analysis, invalid_specs = classify_analysis(
+    data_models, global_analysis, analysis, invalid_specs, queries = classify_analysis(
```

Comment:
This is getting a bit silly :/

Reply:
Alternatively you could define a new dataclass similar to [this](https://github.com/panther-labs/panther/blob/master/internal/log_analysis/rules_engine/src/__init__.py#L47-L52) if you think it would help you manage them. 

```
@dataclass
class Analysis:
    """Contains all the available analysis types"""
    data_models: <Type>
    globals: <type>
    analysis: <type> 
....
```

Reply:
@lindsey-w updated this recently so its a bit more sensible now.